source/libnormaliz/integer.cpp

```diff
@@ -228,9 +228,9 @@ renf_elem_class gcd(const renf_elem_class& a, const renf_elem_class& b){
 #endif
 
 
-template long gcd<long>(const long& a, const long& b);
-template nmz_float gcd<nmz_float>(const nmz_float& a, const nmz_float& b);
-template long long gcd<long long>(const long long& a, const long long& b);
+template <> long gcd(const long& a, const long& b);
+template <> nmz_float gcd(const nmz_float& a, const nmz_float& b);
+template <> long long gcd(const long long& a, const long long& b);
```

Comment:
This silences warnings that say "explicit instantiation of [...] that occurs after an explicit specialization has no effect"